djconnectwise/models.py

```diff
@@ -1181,6 +1212,9 @@ class Ticket(TimeStampedModel):
     project = models.ForeignKey(
         'Project', blank=True, null=True, related_name='project_tickets',
         on_delete=models.CASCADE)
+    phase = models.ForeignKey(
+        'ProjectPhase', blank=True, null=True, related_name='phase_tickets',
```

Comment:
I could be wrong, but I think `just related_name=tickets` would probably be fine here. We shouldn't need to differentiate between phase_tickets and anything else. The related name `tickets` should be fine on all of them and would be a lot cleaner. Don't worry about that in this issue I guess... that might be a change for later. Just wanted to get it down somewhere. 

Leave it as you have it already.